src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -73,6 +74,17 @@
   margin: 0;
 }
 
+.impactTitle {
+  color: var(--color-sheltertech-blue);
+}
+
+.subtitle {
+  font: var(--font-body-medium);
+  grid-column: 2 / span 10;
+  margin: 20px 0 0;
```

Comment:
When you're just setting one margin direction, could you use the individual CSS property (e.g. `margin-top`)? This way, you won't unnecessarily have to specify the margins for the other directions.